storage/storage-transfer/pom.xml

```diff
@@ -51,6 +51,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
One question on directory structure, wdyt about moving these files one directory up outside of the `storage/` directory because it's a different API?